src/SwedbankPay.Sdk.Infrastructure/SwedbankPayClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using SwedbankPay.Sdk.Consumers;
+using SwedbankPay.Sdk.Payments;
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk
+{
+    public class SwedbankPayClient : ISwedbankPayClient
+    {
+        private readonly HttpClient httpClient;
```

Comment:
Codacy found an issue: [Remove the field 'httpClient' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)